doc/source/train/examples/pytorch/pytorch_resnet_finetune.ipynb

```diff
@@ -0,0 +1,776 @@
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023-03-01 12:41:21,195\tWARNING util.py:244 -- The `process_trial_save` operation took 2.936 s, which may be a performance bottleneck.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(RayTrainWorker pid=1478, ip=10.0.62.120) Epoch 9-train Loss: 0.1884 Acc: 0.9344\n",
+      "(RayTrainWorker pid=1478, ip=10.0.62.120) Epoch 9-val Loss: 0.1439 Acc: 0.9737\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023-03-01 12:41:25,360\tWARNING util.py:244 -- The `process_trial_save` operation took 2.947 s, which may be a performance bottleneck.\n",
+      "2023-03-01 12:41:29,205\tWARNING util.py:244 -- The `process_trial_save` operation took 2.696 s, which may be a performance bottleneck.\n",
+      "2023-03-01 12:41:33,757\tINFO tune.py:825 -- Total run time: 66.96 seconds (65.01 seconds for the tuning loop).\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Result(\n",
+      "  metrics={'loss': 0.14393797124686994, 'acc': 0.9736842105263158, 'should_checkpoint': True, 'done': True, 'trial_id': '4c393_00000', 'experiment_tag': '0'},\n",
+      "  log_dir=PosixPath('/tmp/ray_results/finetune-resnet/TorchTrainer_4c393_00000_0_2023-03-01_12-40-31'),\n",
+      "  checkpoint=TorchCheckpoint(local_path=/tmp/ray_results/finetune-resnet/TorchTrainer_4c393_00000_0_2023-03-01_12-40-31/checkpoint_000009)\n",
+      ")\n"
+     ]
+    }
+   ],
+   "source": [
+    "trainer = TorchTrainer(\n",
+    "    train_loop_per_worker=train_loop_per_worker,\n",
+    "    train_loop_config=train_loop_config,\n",
+    "    scaling_config=scaling_config,\n",
+    "    run_config=run_config,\n",
+    ")\n",
+    "\n",
+    "result = trainer.fit()\n",
+    "print(result)\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the checkpoint for batch prediction:\n",
+    "\n",
+    " \n",
+    " The metadata and checkpoints have already been saved in the `local_dir` specified in TorchTrainer:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/tmp/ray_results/finetune-resnet/TorchTrainer_4c393_00000_0_2023-03-01_12-40-31/checkpoint_000009\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "['.metadata.pkl', 'dict_checkpoint.pkl', '.is_checkpoint', '.tune_metadata']"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import os\n",
+    "\n",
+    "checkpoint_folder = result.checkpoint.uri.replace(\"file://\", \"\")\n",
+    "print(checkpoint_folder)\n",
+    "os.listdir(checkpoint_folder)\n"
```

Comment:
Can we do `TorchCheckpoint.from_uri(...)` to handle the `file://`? Also, let's not show the contents of the checkpoint folder - not too useful to see.

Reply:
Ok